go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/golang/mock v1.4.3
 	github.com/golang/protobuf v1.3.2
 	github.com/google/uuid v1.1.1
+	github.com/juju/testing v0.0.0-20201030020617-7189b3728523 // indirect
```

Comment:
where is that dependency coming from? I don't see a reference to it anywhere in `github.com/containernetworking/plugins`. It's not ideal to have it in our go.mod, since it uses an LGPL license. Note that it is not rejected by our license checker because it's not actually used in the final Antrea binaries. So not a big deal. But some license check tools run by companies using Antrea commercially may complain.

Reply:
The dependency was introduced by go-powershell library because it doesn't use go mod and imports `github.com/juju/errors`.
```
# go mod why -m github.com/juju/testing
# github.com/juju/testing
github.com/vmware-tanzu/antrea/pkg/agent/util
github.com/benmoss/go-powershell
github.com/juju/errors
github.com/juju/errors.test
github.com/juju/testing/checkers
```

It appeared as an indirect dependency when bumping up plugins to >=0.8.6 because plugins removed its dependency on juju/testing in 0.8.6: https://github.com/containernetworking/plugins/commit/486ef96e6ffdc33abf7fa711d7a805d147b4d538#diff-3295df7234525439d778f1b282d146a4f1ff6b415248aaac074e8042d9f42d63, so go mod in antrea needs to pin it as go-powershell doesn't pin it.

It seems we might need to clean up some dependency for windows if `juju/errors` doesn't satisfy the license requirement?

Reply:
Thanks for checking what's depending on the `juju` modules. Since the dependency comes from `https://github.com/benmoss/go-powershell` which we only use for Windows, it also explains why the license checker doesn't complain (it only runs on Antrea Linux binaries).

We should definitely look into removing the dependency on `juju`, but I now understand that it is unrelated to this PR. I will open a new issue to track this.